### PR TITLE
Inconsistent content type header

### PR DESCRIPTION
**Pull request by alexanderLavrik**
Set content type returning data from cache

### CODE REVIEW

File: httpmw/ip_cognito_auth.go [hunk 1]
```diff
@@ -128,7 +128,7 @@ type CognitoClaims struct {
 	ClientID string   `json:"client_id"`
 	ISS      string   `json:"iss"`
 	Groups   []string `json:"cognito:groups"`
-}
+}
```
Comment (Drynok) [inline httpmw/ip_cognito_auth.go]: is this whitespace neccesary ?

**Comment by Drynok** [comment]
Looks good

**Comment by prabhat393** [comment]
Looks good.

**Comment by protsack-stephan** [comment]
@alexanderLavrik Can you please add a check for content type in unit tests?

**Comment by alexanderLavrik** [comment]
> @alexanderLavrik Can you please add a check for content type in unit tests?

Done

**Pull request merged**